sqlstore/src/main/resources/db/migration/V8000__archive_library_type.sql

```diff
@@ -0,0 +1,2 @@
+ALTER TABLE LibraryType ADD COLUMN archived boolean DEFAULT FALSE;
+UPDATE LibraryType SET archived = TRUE WHERE platformType = 'Illumina' AND description NOT IN ('Paired End', 'Mate Pair', 'Single End');
```

Comment:
@frankie13 , this will work to keep all the Library Types in the database, but we can now reduce the list shown to the users to only the 3 Types they regularly interact with, so they don't have to scroll through 26 items every time.